application/common/models/Mfa.php

```diff
@@ -55,20 +63,23 @@ public function fields(): array
                 }
                 return null;
             },
-            'data' => function ($model) {
-                $data = [];
-                /** @var Mfa $model */
-                if ($model->verified === 1 && $model->scenario === User::SCENARIO_AUTHENTICATE) {
-                    $data += $model->authInit();
-                }
-                if ($model->type === self::TYPE_BACKUPCODE || $model->type === self::TYPE_MANAGER) {
-                    $data += ['count' => count($model->mfaBackupcodes)];
-                }
-                return $data;
-            }
+            'data',
```

Comment:
Just double checking if this is OK as is?

Reply:
I believe so